integration-tests/olminstall/install/kubeconfig_cluster_label.py

```
@@ -51,7 +112,47 @@ def cluster_label_from_kubeconfig(kubeconfig: Path | str) -> str:
-    return ""
+    return _cluster_label_from_kubeconfig_yaml(path)
+
+
+def _cluster_label_from_cluster_source(
+    cluster_source: str,
+    *,
+    dashboard_url: str = "",
+) -> str:
+    """Derive cluster id from tenant kubeconfig Secret name (``CLUSTER_SOURCE``)."""
+    secret = (cluster_source or "").strip()
+    if not secret or secret == "EAAS":
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "secret"

(S105)